app/assets/javascripts/application.js

```diff
@@ -35,5 +36,7 @@
   //messageBox.scrollTop = messageBox.scrollHeight;
   console.log("inside js")
   //$('ul.messaging-list').scrollTop($('ul.messaging-list li:last-child').position().top);
-
+ $("li.message-display").load(function(){
+   $(".fa-spin").hide();;
+ });
```

Comment:
I was just playing with that and I forgot to comment it out. You are right, nothing happens with that.